docs/Azure.md

```diff
@@ -55,14 +55,14 @@ You'll need the environment variables below:
 Basic usage:
 
 ```sh
-λ> ./tools/azure/getcredentials.sh
-Usage:  ./tools/azure/getcredentials.sh <(sh|nash)> <subscription name> <service principal name> <service secret>
+make credentials sh=(sh|nash) subscription=(subscription) service-principal=(sp name) service-secret=(secret)
 ```
 
 For cool nash shells:
 
 ```sh
 λ> ./tools/azure/getcredentials.sh nash subscription-name klb-sp-tests 123456
+make credentials sh=nash subscription=subscription-name service-principal=klb-sp-tests service-secret=123456
```

Comment:
For nash we need to have double quotes (") around the variable definitions.

Reply:
Indeed, thanks =)